.travis.yml

```diff
@@ -4,9 +4,9 @@ services:
   - docker
 env:
   global:
-    - K8S_VERSION=v1.13.4
-    - MINIKUBE_VERSION=v0.35.0
-    - HELM_VERSION=v2.13.0
+    - K8S_VERSION=v1.18.0
```

Comment:
Maybe try with a least aggressive update? FYI, Google GKE stable is on 1.14'ish. Maybe 1.16.11?

Reply:
I think we need to be mooaaaaarrrr aggressive ;-)